osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/stub/GetActualPowerQualityBundleCommandExecutorStub.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2015 Smart Society Services B.V.
```

Comment:
Copyright statement should probably contain 2021 and Alliander N.V. instead of 2015 and Smart Society Services B.V. as this appears to be a recently added source file.

Reply:
Changed